examples/networking/http_filter/http-parse-complete.c

```diff
@@ -92,7 +92,8 @@ int http_filter(struct __sk_buff *skb) {
 	unsigned long p[7];
 	int i = 0;
 	int j = 0;
-	for (i = payload_offset ; i < (payload_offset + 7) ; i++) {
+	const int last_index = payload_offset + 7;
```

Comment:
This is certainly working before. But llvm compiler loop unrolling has evolved not liking the expression in the for loop conditions. I did not track down why.

Just with "int last_index = payload_offset + 7" also works fine. The point is to have a variable instead of expression.